src/ORM/Behavior/TreeBehavior.php

```diff
@@ -75,6 +75,7 @@ class TreeBehavior extends Behavior
         'scope' => null,
         'level' => null,
         'recoverOrder' => null,
+        'enableOrmDelete' => false,
```

Comment:
I think we would use `use` prefix instead of `enable`.

Reply:
I knew that my naming schema may be off here 😄